test/policies/stripe_customer_policy_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule CodeCorps.StripeCustomerPolicyTest do
+  use CodeCorps.PolicyCase
+
+  import CodeCorps.StripeCustomer
+  import CodeCorps.StripeCustomerPolicy, only: [index?: 2, show?: 2]
+
+  alias CodeCorps.StripeCustomer
+
+  describe "index?" do
+    test "returns true when user is an admin" do
+
+    end
+
+    test "returns true when user is viewing their own information" do
+
+    end
+
+    test "returns false when user id is not the StripeCustomer id" do
+
+    end
+  end
+
+  describe "show?" do
+    test "returns true when user is viewing their own information" do
+
+    end
+
+    test "returns true when user is an admin" do
+
+    end
```

Comment:
Would put `admin` first like in the test above. Think about it as reducing privileges as you go.